app/jobs/create_account_fcrepo_job.rb

```diff
@@ -0,0 +1,6 @@
+# This job is the same as its parent, but chains to the next step
+class CreateAccountFcrepoJob < CreateFcrepoEndpointJob
```

Comment:
@cbeer Yes, `CreateDefaultAdminSetJob` should be idempotent because `AdminSet.find_or_create_default_admin_set_id` is: https://github.com/projecthydra-labs/hyrax/blob/master/app/models/concerns/hyrax/admin_set_behavior.rb#L42-L47